php-frontend/src/main/java/org/sonar/php/metrics/MetricsVisitor.java

```diff
@@ -60,6 +60,8 @@
   private FileMeasures fileMeasures;
   private FileLinesContext fileLinesContext;
 
+  private Map<File, Integer> numberOfLinesOfCode;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1213)